src_c/mixer.c

```diff
@@ -454,7 +454,7 @@ _init(int freq, int size, int channels, int chunk, char *devicename, int allowed
         if (SDL_InitSubSystem(SDL_INIT_AUDIO) == -1)
             return PyInt_FromLong(0);
 
-#if IS_SDLv2
+#ifdef Mix_OpenAudioDevice
```

Comment:
Won't this always be false?

Reply:
Yes, oops. I fixed it in https://github.com/pygame/pygame/pull/1078/commits/1814c96bebc899febd44c6541b5f00a52ae30ccb